linker-common-lib/src/main/java/com/linker/common/MessageContentOutput.java

```diff
@@ -6,12 +6,14 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import java.io.Serializable;
+
 @Getter
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
 @EqualsAndHashCode
-public class MessageContentOutput {
+public class MessageContentOutput implements Serializable {
```

Comment:
If we implement `Serializable`, using `serialVersionUID` is highly recommended.

Reply:
fixed